service/src/main/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsDataService.java

```diff
@@ -231,7 +232,7 @@ private Child buildChild(final ChildParty childParty) {
         return Child.builder()
             .firstName(childParty.getFirstName())
             .lastName(childParty.getLastName())
-            .gender(convertStringToGender(childParty.getGender()))
+            .gender(Optional.ofNullable(childParty.getGender()).map(ChildGender::getRoboticsLabel).orElse(null))
```

Comment:
Sorry, a bit confuse.
In the convertStringToGender() method, it was converting male/female/boy/girl to MALE/FEMALE. It satisfies the ticket requirement already. Why do we need to change it to Boy/Girl?